Continue long argument line to new line to avoid line truncated error

GrantHecht opened the pull request:
The current setup builds fine when built directly on my M1 MacBook Pro. However, I get a line-truncated error in c/bobyqa_c.f90 when building using BinaryBuilder.jl. This change just continues the list of arguments in the call to `bobyqa` to a new line to avoid this error.

vcantarella commented:
Thank you @GrantHecht! Are you also making a pull request to the julia side? Glad to see that, thanks!

GrantHecht commented:
@vcantarella, I'm not sure yet. I believe we've updated v0.8.0 of `prima` in your fork, but it looks like there's been an ongoing discussion since Apr 4, 2024 [here](https://github.com/libprima/PRIMA.jl/pull/29) on how to update PRIMA.jl for the new v0.8.0 API and this discussion seems to have stalled...

That said, I've also updated the prior version of `prima` (with the C API currently used by PRIMA.jl) [here](https://github.com/GrantHecht/prima/tree/add_c_api_bobyqa_honour_x0_option) on branch `add_c_api_bobyqa_honour_x0_option` and updated PRIMA.jl to work with this [here](https://github.com/GrantHecht/PRIMA.jl). I have tested this on my MacBook and it seems to work fine, so you may be able to use my forks for now if you're trying to work in Julia. 

I think ultimately, it would be best to get PRIMA.jl updated to work with the new v0.8.0 C API, but until this is figured out, I think it might be easiest/quickest to suggest also updating the old C API like I've done on my forks. I'm not sure how the prima/PRIMA.jl devs will want to handle this but I'm thinking it might be worth starting a new issue about this to get the ball rolling.
